apis/openstack-swift/src/main/java/org/jclouds/openstack/swift/v1/blobstore/RegionScopedSwiftBlobStore.java

```diff
@@ -476,7 +476,12 @@ public String completeMultipartUpload(MultipartUpload mpu, List<MultipartPart> p
    @Override
    public MultipartPart uploadMultipartPart(MultipartUpload mpu, int partNumber, Payload payload) {
       String partName = getMPUPartName(mpu, partNumber);
+
       String eTag = api.getObjectApi(regionId, mpu.containerName()).put(partName, payload);
+      if (eTag == null) {
```

Comment:
That is the question. From https://issues.apache.org/jira/browse/JCLOUDS-1064

I thought it simply indicated (service-side?) failure, even with no fallback annotation on the swift put.